### ISSUE TITLE
Paged table sorting

### ISSUE THREAD

**Issue by mmcfarland** [WikiWatershed/model-my-watershed#1594]
The paging control on the Analyze tables now increment pages of 50 records at a time.  The sorting mechanism (from bootstrap-table) sorts the actual contents of the rendered table, resulting in sorting _within_ a page.  The sort behavior should still sort the entire collection, rendering the first page of the newly sorted collection/table.

**Comment by arottersman** [comment]
In case it's helpful -- my first thought on this was to sort the Backbone paginated collection on the bootstrap table event `onSort`. I tried this out by calling the `renderBootstrap()` function below wherever we'd usually call `$('[data-toggle="table"]').bootstrapTable()` in the `CatchmentWaterQualityTableView` in `analyze/views.js`

```
    renderBootstrap: function() {
        $('[data-toggle="table"]').bootstrapTable({
            onSort: function(name, order) { // name will be the column's `data-field-name`
                this.collection.setSorting(name, order === 'asc' ? 1 : -1, {full: true});
                this.collection.comparator = this.buildComparator(name);
                this.collection.sort()

                this.collection.setSorting(null);

                this.render()
                this.renderBootstrap();
            }.bind(this));
    },

    buildComparator: function(attributeName) {
        return function(x, y) {
            utils.noDataSort(x.attributes[attributeName],
                y.attributes[attributeName])
        }
    },
```

This did reorder the table, but not in any way I could figure out the logic behind. It definitely didn't sort it.

**Comment by arottersman** [comment]
Why was this issue closed? It doesn't look like it was fixed.

**Comment by rajadain** [comment]
Ugh 🤦‍♂️  I think this line in the #1849 description autoclosed this:

> Now that we're relying more on the pagination, we might have to really fix #1594.

**Comment by arottersman** [comment]
Weird. Do you think it was the word `fix` or any time we mention another issue in a PR it does that? I  feel like I mention  _a lot_ of issues...
